src/controllers/dashboard/encodingsettings.html

```diff
@@ -251,9 +251,8 @@ <h3 class="checkboxListLabel">${LabelHardwareEncodingOptions}</h3>
                 <div class="inputContainer fldEncoderPath">
                     <div style="display: flex; align-items: center;">
                         <div style="flex-grow:1;">
-                            <input is="emby-input" class="txtEncoderPath" label="${LabelffmpegPath}" autocomplete="off" dir="ltr" />
+                            <input is="emby-input" class="txtEncoderPath" label="${LabelffmpegPath}" autocomplete="off" dir="ltr" disabled/>
```

Comment:
Maybe add update the description to inform users that they need to modify the ffmpeg path in the config file if necessary?

Reply:
I think we will want to have this in the changelog.

Currently the ffmpeg path order is command line -> config -> system, and most of our packaging method will use command line to specify ffmpeg so the config file would not have effect and the user will want to change the cli in this case.